Dockerfile

```diff
@@ -0,0 +1,53 @@
+### Note: This DockerFile is adapted from https://github.com/ucb-bar/chipyard/blob/master/.circleci/images/Dockerfile which was adapted from: https://github.com/CircleCI-Public/example-images/openjdk
+# This is a full chipyard setup, which will be built manually on-demand in the Tendermint Hardware Project at https://gitlab.com/virgohardware/core/
+
+FROM ubuntu:18.04
+
+MAINTAINER jacobgadikian@gmail.com
+
+# man directory is missing in some base images
+# https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=863199
```

Comment:
This is not needed anymore since you got rid of the `mkdir -p /usr/share/man/man1`. On a side note, can you confirm that that directory exists.